guides/common/modules/con_acd_architecture.adoc

```diff
@@ -0,0 +1,116 @@
+[id="{context}_architecture"]
+= Architecture
+
+ACD uses {SmartProxies} and a remote execution provider called `acd`.
+
+Ansible playbooks are downloaded from {ProjectServer} to {SmartProxyServer} before being executed.
+You only need to submit the Ansible playbook to the {ProjectServer}, but not an arbitrary number of connected {SmartProxyServer}s.
+
+[id="{context}_ansible_playbooks"]
+== Ansible Playbooks
+
+Ansible playbooks provide the configuration of deployed applications.
+They are required along with to an xref:{context}_application_definitions[application definition] to create xref:{context}_application_instances[application instances].
+
+Ansible group variables are mandatory and only supported as `key:value`, i.e. lists and dictionaries are currently not allowed and cannot be edited via management UI.
+
+Ansible playbooks need to contain Ansible group variables for each provided service.
+
+The Ansible inventory is automatically created by the ACD plugin.
+
+In the {ProjectWebUI}, navigate to *Applications > Ansible Playbooks* to manage Ansible playbooks for application centric deployment.
+
+[CAUTION]
+====
+The Ansible playbook must contain all necessary roles and tasks to execute the playbook.
+
+If your Ansible playbooks are currently using Ansible collections, ensure that these Ansible collections are added manually to every {SmartProxyServer}.
+You can install Ansible collections by hand by running the following command on your {ProjectServer} and {SmartProxyServer}:
```

Comment:
```suggestion
 To install Ansible collections manually, enter the following command on your {ProjectServer} and {SmartProxyServer}:
```
A few things here. 
This "by hand" sounds very strange in English. We generally use the term "manually". 
We always use "enter" for a command and "run" for a script.

Reply:
Thanks, applied!